kolibri/core/device/api.py

```diff
@@ -113,6 +113,19 @@ def get(self, request, format=None):
         info["python_version"] = "{major}.{minor}.{micro}".format(
             major=version_info.major, minor=version_info.minor, micro=version_info.micro
         )
+
+        if not request.user.is_superuser:
+            # If user is not superuser, return just free space available
+            info.pop("version")
```

Comment:
why is `version` restricted? we show it even in the kolibri login page

Reply:
Thanks for catching that! I updated it. Are there any additional keys that non-superadmins need access to?

Reply:
honestly, I think the rest of the keys are not important for non-superadmins, urls might be but the superadmin might want not to show all of them, so I think it's good as it is